processor/src/main/java/org/bindgen/processor/util/ConfUtil.java

```diff
@@ -19,18 +18,18 @@
 		Map<String, String> properties = new LinkedHashMap<String, String>();
 
 		// Eclipse, ant, and maven all act a little differently here, so try both source and class output
-		File file = null;
+		InputStream inputStream = null;
 		for (Location location : new Location[] { StandardLocation.SOURCE_OUTPUT, StandardLocation.CLASS_OUTPUT }) {
-			file = resolveBindgenPropertiesIfExists(location, env, fileName);
-			if (file != null) {
+			inputStream = resolveBindgenPropertiesIfExists(location, env, fileName);
+			if (inputStream != null) {
 				break;
 			}
 		}
 
-		if (file != null) {
+		if (inputStream != null) {
 			Properties p = new Properties();
-			try {
-				p.load(new FileInputStream(file));
+			try (InputStream is = inputStream) {
```

Comment:
Side note: with Java 9+ we could just use `inputStream` instead of using a new local variable (`is`). I will check with @lalmeras if we can upgrade it, any time soon.

Reply:
About 80% of Java developers said in the largest [survey](https://snyk.io/blog/jvm-ecosystem-report-2018) about the JVM Ecosystem that their main projects are running on JDK 8. Keeping Java 8 compatibility is essential in my opinion.